scripts/background/StateManager.js

```diff
@@ -143,6 +145,8 @@ function StateManager() {
 		else if (!isKK()) {
```

Comment:
This has nothing to do with the PR, however, `isKKTime` affects the output of `isKK()` in line 144, as `isKK()` will return `true` if the time is now Saturday 8-12pm, and the corresponding option is enabled. Therefore, this code is required, otherwise K.K. will not automatically start playing once it hits 8pm on a Saturday night.